htdocs/ticket/class/ticket.class.php

```diff
@@ -2812,6 +2812,7 @@ public function sendTicketMessageByEmail($subject, $message, $send_internal_cc =
 
 		$from = $conf->global->TICKET_NOTIFICATION_EMAIL_FROM;
 		$is_sent = false;
+		$array_receiver = $this->removeClosedContact($array_receiver);
```

Comment:
I don't think we must remove after having loaded :
We should modify the  getInfosTicketInternalContact and add a parameter  $onlyopen = 1 for example.
Then we call line 2805    getInfosTicketInternalContact(1)   
Warning is also useless because warning should be used when 
- something is wrong and it is the common life to have contact disabled. 
- an action is expected to fix
Here we should just not include them. No need to alert user to make an action.